test/cypress/support/commands.ts

```diff
@@ -37,8 +38,10 @@ const getCommonHeaders = () => ({
 declare global {
     namespace Cypress {
         // NOTE: necessary for extending the Cypress API
-        // eslint-disable-next-line @typescript-eslint/no-empty-interface
-        export interface Chainable extends GetCypressChainableFromCommands<typeof commands> {}
+        export interface Chainable extends GetCypressChainableFromCommands<typeof commands> {
+            // typing for cypress-get-table
```

Comment:
Optional:

We could use a JSDoc comment here, so the comment appears whenever you hover over `getTable` in tests. We could also enhance the comment by adding a link to the library itself, making it easier to get to :slightly_smiling_face: 

```suggestion
            /**
             * Returns the table data
             *
             * @see https://www.npmjs.com/package/cypress-get-table
             */
```

![image](https://user-images.githubusercontent.com/889383/114143775-d660dd00-9914-11eb-8326-1cb8814c7880.png)